### DISCUSSION TITLE
How to set status using Continuous results uploading

### DISCUSSION THREAD

**Discussion by WillCoutinho** [ludeknovy/jtl-reporter#279] [Q&A] [answered]
Hi there! I'm using Locust with JTL Reporter and I didn't found a way to set result status. 
I'm using this [listener file](https://github.com/ludeknovy/jtl-reporter/blob/main/scripts/jtl_listener_service.py) into the project (as doc suggest), and I tried to set status directly on object "result" in the function "add_result" (line 189).
Can anyone explain how to set properly?

Example of setting the status using API: https://jtlreporter.site/docs/guides/api-data-upload#rest-api

**Comment by ludeknovy** [comment, marked as answer]
Hi @WillCoutinho !
Please have a look at here: https://github.com/ludeknovy/jtl-reporter/issues/276
I will amend the documentation.

**Reply by WillCoutinho** [reply]
Thanks a lot!
